### DISCUSSION TITLE
debugger wont start

### DISCUSSION THREAD

**Discussion by ajr001** [coder/code-server#3849] [Q&A] [answered]
When clicking Run  and debug in my python-enabled workspace, nothing happens except for a progress indicator (blue dots moving across at the top). I expected a panel to pop up asking for the debug config. On a local vscode, I repeated this and created a launch configuration and copied it over to my remote workspace. Now, clicking on Run and debug, shows the debug start button and other debug controls. but when i click start, nothing happens except for the progress indicator again.

**Comment by jsjoeio** [comment]
Hmm...there could be an issue with the debugger. Mind adding a couple steps for me to reproduce locally? I don't write Python very much and haven't used the debugger.

**Comment by ajr001** [comment, marked as answer]
Thanks, I cant repro it any more. I did nothing, the only change I'm aware
of is, today is Tuesday.
:)
